content/en/setup/web-terminal.md

```diff
@@ -5,18 +5,20 @@ type: docs
 onlyWhen: openshift
 ---
 
-## Using the web terminal
-
 Using OpenShift's web terminal might be more convenient for you as it doesn't require you to install `{{% param cliToolName %}}` locally on your computer.
 
+{{% alert title="Note" color="info" %}}
+If you do change your mind, head right over to {{<link "local-usage">}}.
+{{% /alert %}}
+
 
-### {{% task %}} Login on the web console
+## {{% task %}} Login on the web console
 
 First of all, open your browser.
 Then, log in on OpenShift's web console using the URL and credentials provided by your trainer.
 
 
-### {{% task %}} Initialize terminal
+## {{% task %}} Initialize terminal
```

Comment:
e.g. add warning to not choose a project and create a new one in the drop-down to prevent common errors.

Reply:
Good idea, I added a warning.